dev-docs/source/dynamic_api.rst

```diff
@@ -47,10 +56,8 @@ When all viewsets are constructed, reading ``router.urls`` returns all available
 as if it was hard-coded. The ``urls.py`` logic of :mod:`dso_api.dynamic_api.urls` module
 exposes those endpoints to Django.
 
-.. tip::
-    To debug datasets and use their models, you can reuse the router logic
-    which already created those models. The following can be used inside ``./manage.py shell``::
+Serializer Class Layout
+-----------------------
 
-        >>> from dso_api.dynamic_api.urls import router
-        >>> Model = router.all_models["dataset"]["tablename"]
-        >>> Model.objects.all()  # etc..
+Where a "viewset" binds all request handling, the "serializer" defines how the layout of the input/output
```

Comment:
Where a "viewset" binds all request handling, the "serializer" defines _what_ the layout of the input/output